packages/lodestar/src/cli/commands/interop.ts

```diff
@@ -120,19 +126,22 @@ export class InteropCommand implements ICliCommand {
       .then((peerId) => initializePeerInfo(peerId, conf.network.multiaddrs))
       .then((peerInfo) => new NodejsNode({peerInfo, bootnodes: conf.network.bootnodes}));
     const config = options.preset === "minimal" ? minimalConfig : mainnetConfig;
+    const tree = ProgressiveMerkleTree.empty(DEPOSIT_CONTRACT_TREE_DEPTH, new MerkleTreeSerialization(config));
+    let state: BeaconState;
     if (options.quickStart) {
```

Comment:
We should remove quickstart option and use genesisTime/validatorCount separately

Reply:
`quickstart` option also help initialize state from ssz/yml file (see `function quickStartOptionToState(`). If we remove it we'll not be able to do that, can you confirm?

Reply:
rename it than to sszState flag and use just for ssz file loading

Reply:
in fact, this is the only place we call `quickStartOptionToState()` function. If we drop `quickstart` option then this function is maybe redundant.